packages/react-component-library/src/components/RadioCard/RadioCard.test.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult } from '@testing-library/react'
+
+import { withFormik } from '../../enhancers/withFormik'
+import FieldProps from '../../types/FieldProps'
+import FormProps from '../../types/FormProps'
+import { RadioCard } from '.'
+
+describe('RadioCard', () => {
+  let field: FieldProps
+  let form: FormProps
+  let label = ''
+  let radio: RenderResult
+
+  beforeEach(() => {
+    field = {
+      name: 'colour',
+      value: '',
+      onChange: jest.fn(),
+      onBlur: jest.fn(),
+    }
+
+    form = {
+      errors: {},
+      touched: {},
+    }
+  })
+
+  describe('when a field has no errors', () => {
+    beforeEach(() => {
+      form.errors = {}
+    })
+
+    describe('and the field has a label', () => {
+      beforeEach(() => {
```

Comment:
I find this level of `beforeEach` nesting hard to read. I appreciate it has been copied from another test but I feel we should tidy these things up as we get chance.